recipes/mosquitto/conda-forge.yml

```diff
@@ -0,0 +1,4 @@
+build_platform:
+  linux_aarch64: linux_64
```

Comment:
If you're including aarch64 is there a reason to not do ppc64le as well?
```suggestion
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```

Reply:
not really, will add!